site/content/en/guides/consumer/function/export/cloud-build/_index.md

```diff
@@ -19,18 +19,16 @@ mkdir function-export-example
 cd function-export-example
 ```
 
-All commands must be run at the root of this directory.
+**All commands must be run at the root of this directory**.
 
 Use `kpt pkg get` to fetch source files of this tutorial:
 
 ```shell script
-kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/function-export-blueprint exmaple-package
-# Init git
-git init
-git remote add origin https://github.com/<USER>/<REPO>.git
+# Fetch source files
```

Comment:
The steps `git init` and `git remote add` are all kept in other workflow example. Why they are removed for cloudbuild?

Reply:
`gcloud builds submit .` is used to submit a build. So a git repo is not required for Cloud Build.